python/phylanx/ast/physl.py

```diff
@@ -11,6 +11,7 @@
 import numpy as np
 import phylanx.execution_tree
 from phylanx import PhylanxSession
+from phylanx.ast.deploy import *
```

Comment:
Please be as specific while importing as possible, e.g.
```
from phylanx.ast.deploy import deployer
```
That will help avoiding name clashing problems down the road.

Reply:
@hkaiser fixed!